docs/supported-technologies.asciidoc

```diff
@@ -91,7 +91,7 @@ the JVM arguments.
  Starting in version 1.18.0, additional spans are created if the servlet dispatches execution to another servlet through the `forward` or
  `include` APIs, or to an error page.
  See also <<supported-app-servers>>
-|1.0.0
+|1.0.0, jakarta.servlet since 1.28.0
```

Comment:
```suggestion
|1.0.0, `jakarta.servlet` since 1.28.0
```

Reply:
I suggest to also add the version for folks who don't know whether a particular version is Jakarta.
Also applies to the other APIs

```suggestion
|1.0.0, 4.0+ (`jakarta.servlet`) since 1.28.0
```